tests/functional/aws-node-sdk/test/bucket/getBucketEncryption.js

```diff
@@ -0,0 +1,83 @@
+const assert = require('assert');
+const { S3 } = require('aws-sdk');
+
+const checkError = require('../../lib/utility/checkError');
+const getConfig = require('../support/config');
+const BucketUtility = require('../../lib/utility/bucket-util');
+const metadata = require('../../../../../lib/metadata/wrapper');
+const { DummyRequestLogger, makeAuthInfo } = require('../../../../unit/helpers');
+
+const bucketName = 'encrypted-bucket';
+const log = new DummyRequestLogger();
+
+function setEncryptionInfo(info, cb) {
+    metadata.getBucket(bucketName, log, (err, bucket) => {
+        if (err) return cb(err);
+        bucket.setServerSideEncryption(info);
+        return metadata.updateBucket(bucket.getName(), bucket, log, cb);
+    });
+}
+
+describe('aws-sdk test get bucket encryption', () => {
+    let s3;
+
+    before(done => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+        s3 = new S3(config);
+        return done();
+    });
+
+    it('should return NoSuchBucket error if bucket does not exist', done => {
```

Comment:
I'd rather put this in a separate test with a dummy bucket name, than in the `before` section